_posts/2020-11-06-JOIN-2020.md

```diff
@@ -0,0 +1,330 @@
+</tr>
+</tbody>
+</table>
+
+### Anke Maerz
+
+<table>
+<tbody>
+<tr>
+<td><img alt="Anke Maerz" src="/img/2020-11-06-JOIN-2020/speakers/anke-maerz.jpg" width="auto" height="auto" target="_blank" class="image fit"></td>
+<td style="vertical-align:top;"><p>I am Anke and I have a mission in my life: To see and enjoy beauty in life, and to help other people to see and enjoy it, too.</p><p>As a Scrum Master and Agile Coach I live this mission by asking myself every morning THE question: What is it that keeps these amazing people from achieving their goals? – And whatever my answer is, that is where I want to support. Because that is where I can create space for them to see and enjoy the beauty of life.</p><p>This has proven very purposeful to me since 2014, when I first got introduced to Agile – and through various big and huge companies in Belgium as well as Germany.</p><p>Before getting into Agile I’ve been project manager and requirements engineer for 2 years, and before that I studied mathematics and psychology at the university of Tübingen (crazy mix? YES! Beautiful, right?!!)</p><p>Even before that time I’ve already been practicing to be a human, which has proven to be most useful on a personal as well as professional level!</p><p>Company: AgileWorks - Ordina Belgium</p></td>
+</tr>
+</tbody>
+</table>
+
+### Francis Laleman
+
+<table>
+<tbody>
+<tr>
+<td style="vertical-align:top;"><p>Today, I mainly work as a designer of cooperative learning processes and learning cultures. I am a gentle facilitator of (Agile) transformation processes, a train-the-trainer, a cooperative learning facilitator and an Agile coach.
+Having spent so much time in the Middle East and in the Indian subcontinent, I have now been turning my attention to Japan - studying TPS, Lean and Kata, but also the less obvious, such as ikigai, Taoism and Zen Buddhism, or the shuhari learning process in traditional Japanese craftsmanship.
+Most probably, however, you will find me in the garden.</p><p>Company: Agile &amp; Learning Beyond Borders</p></td>
+<td><img alt="Francis Laleman" src="/img/2020-11-06-JOIN-2020/speakers/francis-laleman.jpg" width="auto" height="auto" target="_blank" class="image fit"></td>
+</tr>
+</tbody>
+</table>
+
+### I would like to thank our speakers once more from the whole of Ordina for the interesting sessions they have given.
+
+<br/><br/>
+
+## Tech Track sessions
+
+### Session 1 - Ghelamco alert: Combining Raspberry Pi and AWS Cloud to get the best of both worlds - Bas Moorkens
+
+<div style="position: relative; width: 100%; height: 0; padding-bottom: 55%;">
+    <iframe src="https://www.youtube.com/embed/Iach3_dh83g" width="100%" height="100%;" style="position: absolute; left: 0; top: 0; bottom: 0; right: 0;"></iframe>
+</div>
+
+In this talk we will present our Ghelamco alerting solution which runs on a raspberry PI and is empowered by the AWS cloud to provide additional capabilities.
```

Comment:
I would personally add another break after each YouTube video because the video and text are somewhat glued together now:

![image](https://user-images.githubusercontent.com/1644089/98245257-f14bb180-1f70-11eb-9906-7aa503aa81a6.png)